### PR TITLE
Add config option to disable ranking information for teams.

### PR DESCRIPTION
**Pull request by meisterT**
You likely still want to make the contest non-public to also 'hide' any ranking information from the public.

### CODE REVIEW

File: webapp/src/Twig/TwigExtension.php [hunk 1]
```diff
@@ -156,6 +156,7 @@ public function getGlobals(): array
                                                $this->config->get('data_source') === DOMJudgeService::DATA_SOURCE_CONFIGURATION_AND_LIVE_EXTERNAL,
             'doc_links'                     => $this->dj->getDocLinks(),
             'allow_registration'            => $selfRegistrationCategoriesCount !== 0,
+            'disable_all_ranking'             => $this->config->get('disable_all_ranking'),
```
Comment (nickygerritsen) [inline webapp/src/Twig/TwigExtension.php:159]: Formatting is a bit wrong?

**Review by nickygerritsen — APPROVED** [review]

**Review by eldering — COMMENTED** [review]
A general question: what's the use case that this is based on? And the ranking can still be determined from the scoreboard, right?

**Comment by meisterT** [comment]
> A general question: what's the use case that this is based on?

Online judge for the finals (or similar settings). I rather prefer having this simple config option over implementing the same changes in the WF branch every year.

> And the ranking can still be determined from the scoreboard, right?

If I did it correctly, the scoreboard is not accessible for teams anymore, and the contest should be non-public, so the scoreboard should also not be visible to the public anymore.

File: webapp/templates/partials/scoreboard_table.html.twig [hunk 2]
```diff
@@ -127,16 +135,18 @@
...
-                {# Only print rank when score is different from the previous team #}
-                {% if not displayRank %}
-                    ?
-                {% elseif previousTeam is null or scoreboard.scores[previousTeam.teamid].rank != score.rank %}
-                    {{ score.rank }}
-                {% else %}
-                {% endif %}
-                {% set previousTeam = score.team %}
-            </td>
+            {% if not disable_all_ranking %}
```
[the remark this answers is not in the document]
Comment (meisterT) [reply webapp/templates/partials/scoreboard_table.html.twig]: Done

File: webapp/src/Service/ScoreboardService.php [hunk 3]
```diff
@@ -891,6 +891,7 @@ public function getScoreboardTwigData(
              ],
              'static' => $static,
         ];
+
```
[the remark this answers is not in the document]
Comment (meisterT) [reply webapp/src/Service/ScoreboardService.php]: It was a left-over of some changes that I tried. Reverted.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**